tests/integration/Elasticsearch/Product/ElasticsearchProductTest.php

```diff
@@ -3410,7 +3410,7 @@ private function createData(): void
                 ->property('red', 'color')
                 ->build(),
             (new ProductBuilder($this->ids, 'zanother-product-3b'))
-                ->name('Foo Sti')
+                ->name('Bar Sti')
```

Comment:
one case was this product still in the index somewhow. imo the whole tests need a refactoring, because they all depend on one indexing "test". which makes it hard to reset the whole ES index